.github/workflows/real-js-benchmark.yml

```diff
@@ -0,0 +1,69 @@
+name: Real JS Samples Benchmark
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+      branches:
+      - master
+
+jobs:
+  benchmark:
+    runs-on: [self-hosted, benchmark-pool-1]
```

Comment:
Can you add some kind of comment that describes about this machine,
like, how to setup this machine?  (directories layout, tools, etc)

Reply:
Done, I added a brief text with the expectation in terms of computer spec, security and persistent state.